app/controllers/admin/community_transactions_controller.rb

```diff
@@ -54,7 +54,6 @@ def index
 
     render("index",
       { locals: {
-        show_status_and_sum: @current_community.payments_in_use?,
```

Comment:
Did you check if the payments_in_use? method is needed somewhere else? It feels like legacy logic that shouldn't exist anymore (I can't even guess what's the deciding factor with listing shapes / transaction processes / transaction settings).